app/styles/variables.css

```diff
@@ -1,10 +1,163 @@
 :root {
-  --dark-gray: #666;
-  --off-white: #f0f2f5;
-  --black: #000000f2;
-  --gray: #a9a9a9;
-  --green: #008000;
-  --white: #ffffff;
-  --light-gray: #ccc;
-  --red: #ff0000;
+
+
+  --body-bg-color: white;
+
+  --input-field-border: #666;
+  --input-field-border--error: #e74c3c;
+  --input-field-border--error-border: red;
+  --input-field-border--focus: rgb(0, 81, 255);
+
+  --landing-page--main-heading: black;
+  --landing-page--sub-heading: black;
+
+  --button-get-started--bg:#041484;
+  --button-get-started--text: white;
+
+  --button-copy--bg: wheat;
+  --button-copy--text: #e30062;
+  --button-copy--border: #e30062;
+
+  --button-verify--text: white;
+  --button-identity--text: white;
```

Comment:
I can see that a lot of variables are repeated, you can make it a bit generic so that you don't have to do that

Reply:
Check this RFC https://github.com/Real-Dev-Squad/website-my/issues/311 Second approach.
I made changes according to it. Rohan approved the second approach.

Reply:
okay, makes sense, thanks